src/tests/hw-2-orders.spec.js

```diff
@@ -0,0 +1,178 @@
+      await page.keyboard.press("Enter");
+      //await page.locator(".toast-message").waitFor();
+
+      const toast = await page.locator(".toast.toast-error");
+      await expect(toast.locator(".toast-message")).toHaveText(
+        "Data z ARESu se nepodařilo načíst, vyplňte je prosím ručně"
+      );
+      await expect(toast).toBeVisible();
+      await expect(toast.locator(".toast-close-button")).toBeVisible();
+      await page.locator(".toast.toast-error").waitFor({ state: "visible", timeout: 5000 });
+    });
+
+    test("Check for email error message", async ({ page }) => {
+      await page.getByRole("textbox", { name: "IČO" }).fill(ICO);
+      await page.getByRole("textbox", { name: "Odběratel" }).dblclick();
+      await page.getByText("×Data z ARESu se nepodařilo").waitFor({ state: "visible", timeout: 5000 });
+      await page.getByRole("textbox", { name: "Odběratel" }).fill(clientName);
+      await page.getByRole("textbox", { name: "Úplná adresa" }).fill(address);
+      await page.getByRole("textbox", { name: "Zastoupena - ředitel(ka) školy" }).fill(substituteName);
+      await page.getByRole("textbox", { name: "Jméno a příjmení" }).fill(contactName);
+      await page.getByRole("textbox", { name: "Telefon" }).fill(contactPhone);
+
+      await page.getByRole("textbox", { name: "Email" }).fill("test@test");
+
+      await page.locator("#start_date_1").fill(getDateWithAddedDays(40));
+      await page.locator("#end_date_1").fill(getDateWithAddedDays(47));
+      await page.getByRole("tab", { name: "Příměstský tábor" }).click();
+      await page.locator("select#camp-date_part").selectOption({ label: "Odpolední" });
+      await page.locator("#camp-students").fill("20");
+      await page.locator("#camp-age").fill("7");
+      await page.locator("#camp-adults").fill("2");
+      await page.getByRole("button", { name: "Uložit objednávku" }).click();
+
+      const errorMessage = await page.locator(".toast-message");
+      await expect(errorMessage).toHaveText("Některé pole obsahuje špatně zadanou hodnotu");
+      const errorTitle = await page.locator(".toast-title");
+      await expect(errorTitle).toHaveText("Špatně zadané pole");
+
+      const invalidFeedbackMessage = await page.locator("span.invalid-feedback");
+      await expect(invalidFeedbackMessage).toHaveText("Zadaná adresa neexistuje, zkontrolujte překlepy");
+
+      await expect(page.locator("h3")).not.toHaveText("Děkujeme za objednávku");
+    });
+
+    test("Should not create an order without fill all fields", async ({ page }) => {
+      await page.getByRole("textbox", { name: "IČO" }).fill(ICO);
+      await page.getByRole("textbox", { name: "Odběratel" }).dblclick();
+      await page.getByRole("tab", { name: "Škola v přírodě" }).click();
+      await page.getByRole("button", { name: "Uložit objednávku" }).click();
+      await expect(page).toHaveURL("https://team8-2022brno.herokuapp.com/objednavka/pridat");
```

Comment:
await expect(page).toHaveURL(`${baseUrl}/objednavka/pridat`);
Lepsi, jelikoz baseUrl se muze menit, nebo budes chtit targetovat vice prostredi a to bude variabilni.